lucene/core/src/test/org/apache/lucene/search/TestLRUQueryCache.java

```diff
@@ -56,6 +59,7 @@
 import org.apache.lucene.index.Term;
 import org.apache.lucene.search.BooleanClause.Occur;
 import org.apache.lucene.store.Directory;
+import org.apache.lucene.util.BytesRef;
```

Comment:
Have you seen the `ant precommit` failure? It complains that this import is unused.

Reply:
@jpountz I am very sorry. I have fixed the above problem and now the PR has passed ant precommit.